code/game/objects/structures/railings.dm

```diff
@@ -17,6 +17,21 @@
 	density = FALSE
 	climbable = FALSE
 
+/obj/structure/railing/cap //aestetic "end" for railing
+	icon_state = "railing_cap"
+	density = FALSE
+	climbable = FALSE
+
+/obj/structure/railing/cap/normal
+	icon_state = "railing_cap_normal"
+	density = FALSE
+	climbable = FALSE
+
+/obj/structure/railing/cap/reversed
+	icon_state = "railing_cap_reversed"
+	density = FALSE
+	climbable = FALSE
```

Comment:
```suggestion
/obj/structure/railing/cap/normal
	icon_state = "railing_cap_normal"

/obj/structure/railing/cap/reversed
	icon_state = "railing_cap_reversed"
```
No need to repeat those, they already inherit it from the /obj/structure/railing/cap